src/bootstrap/flags.rs

```diff
@@ -75,7 +75,22 @@ pub enum Subcommand {
 
 impl Flags {
     pub fn parse(args: &[String]) -> Flags {
+        let mut extra_help = String::new();
```

Comment:
Oh I see.  What do you suggest would make it clearer?  I haven't observed a standard help message that indicates such a capability, rather it seems to be generally assumed.  My first inclination would be to annotate the "Options:" header in some way, but that header is part of the generated help string by getopts in this instance.